changesets/scripts/entrypoint.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+search_key=$1
+search_value=$2
+cached=$3
+modify_string="hgv=no to hgv=yes"
+
+if [ "$cached" = "true" ]; then
+  printf "Using existing files/data.osm file\n"
+else
+  printf "Downloading osm data\n"
+  wget http://download.geofabrik.de/north-america/us/massachusetts-latest.osm.pbf -O files/data.osm.pbf
+
+  printf "Converting osm.pbf file to osm format\n"
+  osmconvert files/data.osm.pbf > files/data.osm
+fi
+
+printf "Filtering OSM data for ways with $search_key=$search_value\n"
+osmfilter files/data.osm --ignore-dependencies --keep= --keep-ways="hgv=no and $search_key=$search_value" > tmp/filtered.osm
+
+printf "Modifying filtered OSM tags with $modify_string\n"
+osmfilter tmp/filtered.osm --modify-tags="$modify_string" > tmp/modified.osm
+
+printf "Updating version number of modified ways\n"
+while IFS= read -r line; do
+  if echo "$line" | grep -q "<way"; then
+    version=$(echo "$line" | grep -o 'version="[0-9]*"' | grep -o '[0-9]*')
+    if [ -n "$version" ]; then
+      new_version=$((version + 1))
+      line=$(echo "$line" | sed "s/version=\"$version\"/version=\"$new_version\"/")
+    fi
+  fi
+  echo "$line"
+done < tmp/modified.osm > tmp/version_updated.osm
+
+
+timestamp=$(date +%s)
```

Comment:
suggestion: for human readable filenames, can we use ISO:8601 here instead?
```suggestion
timestamp=$(date -Iseconds)
```
or
```suggestion
timestamp=$(date -Iseconds --universal)
```

Reply:
thoughts on something like this instead? 
```
$ date +%Y%m%dT%H%M%S  
> 20250520T132944
```
`20250520T132944_changeset.osc`
vs 
`2025-05-20T19:28:04+00:00_changeset.osc`

Reply:
I prefer standardized date strings, I just don't personally find it worth the time to create custom date strings. So I wouldn't go with that over ISO8601, is there a reason you don't like ISO8601? AFAIK I think all the characters are compatible with git/filesystems?

Reply:
for me it's just harder to parse, especially w/ the time formatted that way